exosphere-shared/src/docker-helper.ls

```diff
@@ -83,18 +83,6 @@ class DockerHelper
         | otherwise => console.log "'#{image}' download complete"; done!
 
 
-  # Runs cat on file in Docker container to print its content
-  @cat-file = ({image, file-name}, done) ->
-    DockerHelper.pull-image {image}, (err) ->
-      | err       => done err
-      | otherwise =>
-        stdout-stream = new stream.PassThrough
-        text-stream-search = new TextStreamAccumulator stdout-stream
-        docker.run image, ['cat', file-name], stdout-stream, (err, data, container) ->
-          | err       => done err
-          | otherwise => done null, text-stream-search.to-string!
-
-
```

Comment:
Ah, I was wondering how tests were still passing. Great, thanks for noticing this!